resources/tests.mjs

```diff
@@ -13,8 +13,8 @@ Suites.push({
         new BenchmarkTestStep('Adding' + numberOfItemsToAdd + 'Items', page => {
             const newTodo = page.querySelector(".new-todo");
             for (let i = 0; i < numberOfItemsToAdd; i++) {
-                newTodo.value = 'Something to do ' + i;
-                newTodo.change();
+                newTodo.setValue('Something to do ' + i);
+                newTodo.dispatchEvent("change");
```

Comment:
Hm... maybe we should stick with ' instead of " for self consistency.

Reply:
Done in PR 26